draft-ietf-quic-multipath.md

```diff
@@ -1199,26 +1297,23 @@ PATH_AVAILABLE frames are formatted as shown in {{fig-path-available-format}}.
 
 PATH_AVAILABLE frames contain the following fields:
 
-Destination Connection ID Sequence Number:
-: The sequence number of the Destination Connection ID used by the
+Path Identifier:
```

Comment:
Typo/missed change: I think the `Destination Connection ID Sequence Number` in the frame spec above needs to be replaced with `Path Identifier`?

Reply:
Done. Thanks!